teamscale_client/client.py

```diff
@@ -45,10 +49,12 @@ def check_api_version(self):
         """
         url = self.get_global_service_url('service-api-info')
         response = self.get(url)
-        apiVersion = response.json()['apiVersion']
-        if apiVersion < 3:
+        json_response = response.json()
+        self.supports_at_least_api_version_5 = int(json_response['minSupportedApiVersion']) >= 5
```

Comment:
I don't think we should implement this level of backwards compatibility. Just make the minium required version `5` and always use the newest API